src/plugins/tune/tune_impl.cpp

```diff
@@ -113,38 +114,29 @@ void TuneImpl::play_tune_async(
 
     LogDebug() << "About to send tune: " << tune_str;
 
-    if (tune_str.size() > 230) {
-        report_tune_result(callback, Tune::Result::TUNE_TOO_LONG);
+    if (tune_str.size() > 248) {
+        report_tune_result(callback, Tune::Result::TuneTooLong);
         return;
     }
 
-    std::string tune1_str;
-    std::string tune2_str;
-
-    if (tune_str.size() < 30) {
-        tune1_str = tune_str;
-        tune2_str = "";
-    } else {
-        tune1_str = tune_str.substr(0, 30);
-        tune2_str = tune_str.substr(30);
-    }
+    const uint32_t TUNE_FORMAT_QBASIC1_1 = 1;
```

Comment:
@julianoes: Is that the right way to do it, or is there a mavlink-generated enum I missed?

Reply:
Yes, there is this one: https://github.com/mavlink/c_library_v2/blob/45d924df376c00e96d40a567c94f111ffc3b9bc1/common/common.h#L1889